sdk/servicebus/service-bus/test/batchReceiver.spec.ts

```diff
@@ -104,7 +104,7 @@ async function beforeEachTest(
 async function afterEachTest(): Promise<void> {
   await ns.close();
 }
-describe("Batch Receiver - Settle message", function (): void {
+describe("Batch Receiver - Settle message", function(): void {
```

Comment:
Prettier extension formats with a space in between `function` and `()`

Reply:
Yes, let's make sure this PR agrees with prettier default config so we can be sure to be done with this forever!

Reply:
Discussed offline.

@HarshaNalluru was testing on a different project where prettier (the npm package) was not installed.

In the service-bus project where the prettier package is installed, it can be seen that it indeed removes the space